app/views/GovukWrapper.scala.html

```diff
@@ -73,8 +73,8 @@
 
 @bodyEnd = {
     @footer(
-        analyticsToken = None,
-        analyticsHost = appConfig.analyticsHost,
+        analyticsToken = Some(appConfig.analyticsToken),
+        analyticsHost = ""
```

Comment:
you're missing a comma at the end of this line

Reply:
Updated. Thanks